hiv-benchmarking/README.md

```diff
@@ -0,0 +1,86 @@
+## HIV benchmarking
+
+This folder contains raw data, processed data, and SPRAS results pertaining to the dataset taken from the following research article: **HIV-1 virological synapse formation enhances infection spread by dysregulating Aurora Kinase B** - Bruce JW, Park E, Magnano C, Horswill M, Richards A, Potts G, et al. (2023)
+
+The data is about human immune cells responding to viral infection and contains data from protein abundance and phosphorylation changes experiments, which will be the input to pathway reconstruction.
```

Comment:
```suggestion
The study examines human immune cells responding to viral infection as well as the changes that take place inside the already infected cells, which is the focus here.
The data is from protein abundance and phosphorylation experiments, which will be the input to pathway reconstruction.
```

We should also describe the two time points because that will be important for understanding the 5 and 60 file names.